src/test/java/org/opensearch/neuralsearch/plugin/NeuralSearchTests.java

```diff
@@ -42,15 +42,15 @@ public void testQueryPhaseSearcher() {
         assertNotNull(queryPhaseSearcher);
         assertTrue(queryPhaseSearcher.isEmpty());
 
-        System.setProperty(NEURAL_SEARCH_HYBRID_SEARCH_DISABLED.getValue(), "true");
+        System.setProperty(NEURAL_SEARCH_HYBRID_SEARCH_ENABLED, "true");
```

Comment:
should we set this property in base class?

and why test are extending OpenSearchTestCase, there is should be a base Neural Search test class

Reply:
I was thinking that for some unit tests we may want to disable it, that would be complex if it's set in base class. 

I'll make setting in a method and will call it in a setup phase for this test class.

Reply:
sure